contentcuration/contentcuration/frontend/channelEdit/components/QuickEditModal/EditBooleanMapModal.vue

```diff
@@ -90,22 +92,41 @@
          * Where nodeIds is the id of the nodes that have the option selected
          */
         selectedValues: {},
-        changed: false,
       };
     },
     computed: {
-      ...mapGetters('contentNode', ['getContentNodes']),
+      ...mapGetters('contentNode', ['getContentNodes', 'getContentNode']),
       nodes() {
         return this.getContentNodes(this.nodeIds);
       },
       isTopicSelected() {
         return this.nodes.some(node => node.kind === ContentKindsNames.TOPIC);
       },
+      canSave() {
+        if (this.hasMixedCategories) {
+          return Object.values(this.selectedValues).some(value => value.length > 0);
+        }
+        return !this.error;
+      },
+      hasMixedCategories() {
```

Comment:
We should not have this method as a computed property, because while we are editing the nodes, the `selectedValues` map will change, and eventually we will have `hasMixedCategories = false` when thats not true, since the persisted values are indeed mixed. So its better to just determine this once, in the `created` hook, when we calculate the optionsNodes [here](https://github.com/AllanOXDi/studio/blob/35dcb4a66365c423da0d8b0e0c2f5a16f6fedf00/contentcuration/contentcuration/frontend/channelEdit/components/QuickEditModal/EditBooleanMapModal.vue#L144), so in this created hook we can have something like:

```javascript
      this.selectedValues = optionsNodes;
      this.hasMixedCategories = Object.values(this.selectedValues).some(
        (value) => value.length < this.nodes.length
      );
```

So this value is never recalculated and always remain the same state as the actual nodes values.